.travis.yml

```diff
@@ -1,8 +1,25 @@
 services:
-    - docker
+- docker
 
-script: docker-compose run test
+script: docker-compose run windbreaker npm run test:docker
+
+after_success:
+  # windbreaker/windbreaker:latest is automatically built and tagged
+  # from test run
+  |
+    if [ $TRAVIS_BRANCH == "master" ]; then
+      docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+      docker tag windbreaker/windbreaker:latest windbreaker/windbreaker:build-$TRAVIS_BUILD_NUMBER;
+
+      # push both the build tag and update latest
+      docker push windbreaker/windbreaker:build-$TRAVIS_BUILD_NUMBER;
+      docker push windbreaker/windbreaker:latest;
```

Comment:
`latest` tag needs to be updated everytime. At the moment we will use the build number to keep track of tags but we can start introducing semver in the future.